readme.txt

```diff
@@ -79,7 +48,6 @@ See also <a href="https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTIN
 
 == Changelog ==
 
-
```

Comment:
I think this changelog area is generated and not written, the changes here might get overiden on each release.

Reply:
:+1: